test/functional/helpers.lua

```diff
@@ -367,7 +367,7 @@ end
 local function set_shell_powershell()
   source([[
     set shell=powershell shellquote=\" shellpipe=\| shellredir=>
-    set shellcmdflag=\ -NoLogo\ -NoProfile\ -ExecutionPolicy\ RemoteSigned\ -Command
+    let &shellcmdflag = '-NoLogo -NoProfile -ExecutionPolicy RemoteSigned -Command Remove-Item alias:cat;'
     let &shellxquote=' '
```

Comment:
Should this be removed? Or should it be set to empty?

I don't remember why I added it but I think it was based on one of our previous discussions in one of the Windows PRs.

Reply:
https://github.com/neovim/neovim/pull/6359#issuecomment-289250959

Reply:
@janlazo comment above is about:

    let &shellxquote=' '
 
I just noticed it and wasn't sure if it affected what you're working on here.

Reply:
> Does the space make a difference?

I don't know, I copied it from https://github.com/neovim/neovim/pull/6359#issuecomment-289250959 . If it doesn't impede your goals then we can leave it.